pageserver/client_grpc/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "pageserver_client_grpc"
+version = "0.1.0"
+edition = "2024"
```

Comment:
We should use the workspace edition: `edition.workspace = true`. We'll also need to use the workspace license: `license.workspace = true`.